### PR TITLE
Added CrossOrigin annotation to enable CORS

### PR DESCRIPTION
**Pull request by hans0**

### CODE REVIEW

File: HeroArena/src/main/java/com/revature/controller/LoginController.java [hunk 1]
```diff
@@ -4,13 +4,20 @@
 
 import org.springframework.beans.factory.annotation.Autowired;
 import org.springframework.stereotype.Controller;
-import org.springframework.web.bind.annotation.*;
+<<<<<<< HEAD
+=======
```
Comment (LuisV) [inline HeroArena/src/main/java/com/revature/controller/LoginController.java]: Why is this here? This seems to be part of a merge conflict that wasn't properly resolved?

Comment (hans0) [reply HeroArena/src/main/java/com/revature/controller/LoginController.java]: Sorry, removed it.

**Comment by hans0** [comment]
It's fixed.

File: HeroArena/src/main/java/com/revature/controller/LoginController.java (hunk 1 above)
Comment (LuisV) [reply HeroArena/src/main/java/com/revature/controller/LoginController.java]: Please commit again.

Comment (hans0) [reply HeroArena/src/main/java/com/revature/controller/LoginController.java]: it's been pushed

Comment (hans0) [reply HeroArena/src/main/java/com/revature/controller/LoginController.java]: ok, dont know why it went back
